.github/workflows/pr.yml

```diff
@@ -0,0 +1,82 @@
+name: pr
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
+
+# Cancel any current or previous job from the same PR
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  tests-gradle:
+    runs-on: macos-11
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build with Gradle
+        run: |
+          ulimit -c unlimited
+          ./gradlew ciTestsGradle
+      - name: Collect Diagnostics
+        if: always()
+        run: ./scripts/collect-diagnostics.main.kts
```

Comment:
Sorry I have a silly question: how come running Kotlin scripts works without needing any setup?

Reply:
This is thanks to @LouisCAD :) https://github.com/actions/virtual-environments/issues/3687